alignak/objects/config.py

```diff
@@ -1749,38 +1749,38 @@ def fill_default_satellites(self):
                                  'manage_arbiters': '1'})
             self.brokers = BrokerLinks([broker])
 
-    def got_broker_module_type_defined(self, mod_type):
+    def got_broker_module_type_defined(self, module):
         """Check if a module type is defined in one of the brokers
 
-        :param mod_type: module type to search
-        :type mod_type: str
+        :param module: module type to search
+        :type module: str
         :return: True if mod_type is found else False
         :rtype: bool
         """
         for broker in self.brokers:
             for module in broker.modules:
-                if hasattr(module, 'module_type') and module.module_type == mod_type:
+                if hasattr(module, 'module') and module.module == module:
```

Comment:
it's the 'name' of module to import, have you another string name for this? I haven't idea of a good name for this